mods/creative/init.lua

```diff
@@ -65,7 +65,10 @@ function creative_inventory.update(player_name, filter, tab_id)
 	for name, def in pairs(tab_category(tab_id)) do
 		if not (def.groups.not_in_creative_inventory == 1) and
 				def.description and def.description ~= "" and
-				(not filter or def.name:find(filter, 1, true)) then
+				(not filter or
+					def.name:find(filter, 1, true) or
+					def.description:lower():find(filter, 1, true)
```

Comment:
You don't need to use `lower() here`, as the string is already lowered through the fields handler : https://github.com/minetest/minetest_game/blob/master/mods/creative/init.lua#L189